pkg/asset/ignition/node.go

```diff
@@ -91,3 +92,48 @@ func ConvertToAppendix(file *igntypes.File) {
 	file.Contents = igntypes.Resource{}
 	file.Overwrite = ignutil.BoolToPtr(false)
 }
+
+// AppendVarPartition appends a /var partition to the ignition configuration to avoid growfs.
+func AppendVarPartition(config *igntypes.Config) {
+	// See https://docs.fedoraproject.org/en-US/fedora-coreos/storage/#_setting_up_separate_var_mounts
+	config.Storage.Disks = append(config.Storage.Disks, igntypes.Disk{
+		Device: "/dev/disk/by-id/coreos-boot-disk",
+		Partitions: []igntypes.Partition{
+			{
+				Label:  ptr.To("root"),
+				Number: 4,
+				// Allocate at least 8GiB to the rootfs.
+				// See https://docs.fedoraproject.org/en-US/fedora-coreos/storage
+				SizeMiB: ptr.To(8192),
+				Resize:  ptr.To(true),
+			},
+			{
+				Label:   ptr.To("var"),
+				SizeMiB: ptr.To(0),
+			},
```

Comment:
Instead of this, follow the layout documented in the OCP docs (i.e. no `root` entry, but add `StartMiB` to the `var` entry).

Reply:
Done.